index.js

```diff
@@ -329,6 +329,19 @@
 
   var def = $.create(checkTypes, env);
 
+  var prop =
+  def('prop',
+      {},
+      [$.String, a, b],
+      function(key, obj) {
+        if (obj != null && key in Object(obj)) {
+          return obj[key];
+        } else {
+          throw new TypeError('‘prop’ expected object to have a property ' +
+            'named ‘' + key + '’; ' + R.toString(obj) + ' does not');
+        }
+      });
+
```

Comment:
Let's define this after defining `method`. It makes sense for the definitions of `def` and `method` to live together, as they are closely related functions.